using-mobile-extensions/adobe-analytics-edge/analytics-edge-api-reference.md

```diff
@@ -0,0 +1,1249 @@
+# Analytics Edge API reference
+
+{% hint style="info" %}
+
+The following Analytics API are currently **not** supported with Analytics Edge. They are listed for reference only.
```

Comment:
We should move trackAction and trackState APIs on top and create a table/list with all the APIs which are not supported below (just API name, no samples). Also, I think we can remove the sample code for the other platforms other than ios/android since it is not supported.